servers/xr/xr_interface_extension.h

```diff
@@ -102,13 +102,19 @@ class XRInterfaceExtension : public XRInterface {
 	virtual Transform3D get_transform_for_view(uint32_t p_view, const Transform3D &p_cam_transform) override;
 	virtual Projection get_projection_for_view(uint32_t p_view, double p_aspect, double p_z_near, double p_z_far) override;
 	virtual RID get_vrs_texture() override;
```

Comment:
If this works with OpenGL, does it make sense that it returns a RID ? Should it not return instead a Variant? As OpenGL may need to return a GLInt.

Reply:
No I think these should be RIDs, we're returning the texture object in the rendering server. Even in rendering device I encapsulate the internal texture objects we get from OpenXR in a texture object so we can store size and formatting and expose the texture to the rest of the engine.